resources/buildConfigDefinitions.js

```diff
@@ -0,0 +1,221 @@
+function last(array) {
+  return array[array.length - 1];
+}
+
+const parsers = require('../src/Options/parsers');
+
+const letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+function toENV(key) {
+  var str = '';
+  let previousIsUpper = false;
+  for(let i = 0; i < key.length; i++) {
+    const char = key[i];
+    if (letters.indexOf(char) >= 0) {
+      if (!previousIsUpper) {
+        str += "_"
+        previousIsUpper = true;
+      }
+    } else {
+      previousIsUpper = false;
+    }
+    str += char;
+  }
+  return str.toUpperCase();
+}
+
+function getCommentValue(comment) {
+  if (!comment) { return }
+  return comment.value.trim();
+}
+
+function getENVPrefix(iface) {
+  if (iface.id.name === 'ParseServerOptions') {
+    return 'PARSE_SERVER_';
+  }
+  if (iface.id.name === 'CustomPagesOptions') {
+    return 'PARSE_SERVER_CUSTOM_PAGES_';
+  }
+  if (iface.id.name === 'LiveQueryServerOptions') {
+    return 'PARSE_SERVER_LIVE_QUERY_';
+  }
+}
+
+function processProperty(property, iface) {
+  const firstComment = getCommentValue(last(property.leadingComments || []));
+  const lastComment = getCommentValue((property.trailingComments || [])[0]);
+  const name = property.key.name;
+  const prefix = getENVPrefix(iface);
+
+  if (!firstComment) {
+    return;
+  }
+  const components = firstComment.split(':ENV:').map((elt) => {
+    return elt.trim();
+  });
+  let defaultValue;
+  if (lastComment && lastComment.indexOf('=') >= 0) {
+    const slice = lastComment.slice(lastComment.indexOf('=') + 1, lastComment.length).trim();
+    defaultValue = slice;
+  }
+  const help = components[0];
+  const env = components[1] || (prefix + toENV(name));
+  let type = property.value.type;
+  let isRequired = true;
+  if (type == 'NullableTypeAnnotation') {
+    isRequired = false;
+    type = property.value.typeAnnotation.type;
+  }
+  return {
+    name,
+    env,
+    help,
+    type,
+    defaultValue,
+    types: property.value.types,
+    typeAnnotation: property.value.typeAnnotation,
+    required: isRequired
+  };
+}
+
+
+function doInterface(iface) {
+  return iface.body.properties.map((prop) => {
+    return processProperty(prop, iface);
+  }).filter((e) => e !== undefined);
+}
+
+function mapperFor(elt, t) {
+  const p = t.identifier('parsers');
+  const wrap = function(identifier) {
+    return t.memberExpression(p, identifier);
```

Comment:
`wrap = (identifier) => t.memberExpression(p, identifier);`

right?  if you prefer?